### PR TITLE
Fix mlserver default metrics port in serving runtime

### PR DESCRIPTION
**Pull request by sivanantha321**
**What this PR does / why we need it**:
From mlserver 1.1.0 metrics port is changed to 8082. Before the metrics were served at port 8080.
Screenshot from 2023-12-09 14-44-24

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Type of changes**
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:

```release-note

```

### CODE REVIEW

File: charts/kserve-resources/templates/clusterservingruntimes.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ metadata:
 spec:
   annotations:
     # mlserver version 1.1.0 uses port 8082 as default instead of 8080.
-    prometheus.kserve.io/port: '8080'
+    prometheus.kserve.io/port: '8082'
```
Comment (yuzisun) [inline charts/kserve-resources/templates/clusterservingruntimes.yaml:41]: How do we make this backwards compatible, as this will break other versions.

**Review by terrytangyuan — COMMENTED** [review]
Is this still needed? We'll be removing MLServer in https://github.com/kserve/kserve/pull/3443

**Comment by sivanantha321** [comment]
If we are removing mlserver then, No

**Comment by sivanantha321** [comment]
Closing this as `mlserver` will be deprecated in future.

**Pull request closed without merging**